specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1801 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-11-01",
+    "title": "AppServiceCertificateOrders API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CertificateRegistration/certificateOrders": {
+      "get": {
+        "tags": [
+          "AppServiceCertificateOrders"
+        ],
+        "summary": "List all certificate orders in a subscription.",
+        "description": "Description for List all certificate orders in a subscription.",
+        "operationId": "AppServiceCertificateOrders_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AppServiceCertificateOrderCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List App Service Certificate orders by subscription": {
+            "$ref": "./examples/ListAppServiceCertificateOrdersBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CertificateRegistration/validateCertificateRegistrationInformation": {
+      "post": {
+        "tags": [
+          "AppServiceCertificateOrders"
+        ],
+        "summary": "Validate information for a certificate order.",
+        "description": "Description for Validate information for a certificate order.",
+        "operationId": "AppServiceCertificateOrders_ValidatePurchaseInformation",
+        "parameters": [
+          {
+            "name": "appServiceCertificateOrder",
```

Comment:
>appServiceCertificateOrder [](http://example.com/codeflow?start=21&length=26)

This parameter must be in the path. IOW, POST actions are taken on resource instances, not on types (or global, generally). POST action path should be this:

"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CertificateRegistration/certificateOrders/{certificateOrderName}/validate", without a request body.